hack/dailybuild/dailybuild.go

```diff
@@ -165,69 +175,105 @@ func uploadAll(tag, gcpToken string) (*Builds, error) {
 	return builds, nil
 }
 
-func createRssFeed(builds *Builds) error {
-	// update rss
-	now, err := time.Parse(time.RFC3339, "2013-01-16T21:52:35-05:00")
+func updateReadme(builds *Builds) error {
+	readme, err := os.ReadFile(fullPathReadme)
 	if err != nil {
-		fmt.Printf("time.Parse failed: %v\n", err)
+		fmt.Printf("ReadFile returned error: %v\n", err)
```

Comment:
Would it be better to do something like `"Failed to parse README file at %s: %v\n", fullPathReadme, err`

Reply:
Will follow up on this for Monday. Going to merge this so we have a build tonight. Will fix this plus I need to delete the rss.xml at the root of the repo.